### PR TITLE
Insertion sort

### LINKED ISSUE #928: Add readme.md for Insertion sort
@jainaman224 can i work on this issue??

### PR DESCRIPTION
**Pull request by vishnu0179**
Fixes #928 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Added README.md for Insertion Sort
- 

#### Languages Used:
- 
- 

#### Files Added:
- README.md
-

### CODE REVIEW

File: Insertion_Sort/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+# INSERTION SORT
```
Comment (jainaman224) [inline Insertion_Sort/README.md]: I think it can be elaborated further.

**Comment by amaaniqbal** [comment]
@vishnu0179 it seems you only wish to update the readme for the insertion sort from your PR heading. If that's the case please update the heading of this PR and incorporate changes suggested by Aman. 

You can update the Readme by adding 1-2 more relevant examples(of edge cases will be recommended) 
A sample readme which you may follow could be https://github.com/jainaman224/Algo_Ds_Notes/tree/master/Heap_Sort

**Comment by MastersAbh** [comment]
@vishnu0179 still working on it ?

**Comment by MastersAbh** [comment]
You might close this due to inactivity @jainaman224

**Comment by rishabhgarg25699** [comment]
@vishnu0179 @jainaman224 I am closing this issue due to inactivity of PR owner .

**Pull request closed without merging**